Meadowlark-js/packages/meadowlark-core/src/message/UpdateRequest.ts

```diff
@@ -3,17 +3,6 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-import { DocumentInfo } from '../model/DocumentInfo';
-import { ResourceInfo } from '../model/ResourceInfo';
-import { Security } from '../security/Security';
+import { UpsertRequest } from './UpsertRequest';
 
-export type UpdateRequest = {
-  meadowlarkId: string;
-  documentUuid: string;
-  resourceInfo: ResourceInfo;
-  documentInfo: DocumentInfo;
-  edfiDoc: object;
-  validate: boolean;
-  security: Security;
-  traceId: string;
-};
+export type UpdateRequest = UpsertRequest;
```

Comment:
They are the same shape, and have been for a long time. Keeping separate names in case they become different in the future.